data/mods/Aftershock/effects.json

```diff
@@ -97,6 +97,7 @@
     "type": "effect_type",
     "id": "afs_isohypsa_overwatch",
     "name": [ "Isohypsa Overwatch" ],
+    "//~": "This is just a fragment of machine code, it needs no translation.",
```

Comment:
This can only be extracted when it goes together with the string, for example `{ "//~": "comments", "str": "text" }`.